remote/index.js

```diff
@@ -66,8 +68,8 @@ module.exports = function (app) {
       vid: app._backstop.testCtr
     };
 
-    const command = req.path.includes('dtest') ? 'test' : 'reference';
-    backstop(command, { config, i: Boolean(req.body.i) }).then(
+    const command = req.path.includes(`/dtest/`) ? 'test' : 'reference';
```

Comment:
Would you mind instead checking for `/dref/` so that default behavior is `test`?  I know it's not currently a concern, but we may alias more routes to this flow.

Reply:
I've updated the PR accordingly.